### ISSUE TITLE
System.ArgumentOutOfRangeException: 该值必须大于或等于零，且必须小于控制台缓冲区在该维度的大小。

### ISSUE THREAD

**Issue by flashlxy** [PowerShell/PSReadLine#1041]
Environment data
----------------

Steps to reproduce or exception report
--------------------------------------
哎呀，出现问题。请在报告此 Bug 时添加以下详细信息。
在 GitHub 上报告: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
上 200 个密钥:
 x e ' Space ' c : \ U s e r s \ f l a s h \ . v s c o d e \ e x t e n s i o n s \ m s - p y t h o n . p y t h o n - 2 0 1 9 . 9 . 3 4 9 1 1 \ p y t h o n F i l e s \ p t v s d _ l a u n c h e r . p y ' Space ' - - d e f a u l t ' Space ' - - c l i e n t ' Space ' - - h o s t ' Space ' l o c a l h o s t ' Space ' - - p o r t ' Space ' 2 4 5 7 1 ' Space ' d : \ C O D E S \ G r i d D r a w \ M i c a p s D a t a D r a w . p y ' Space Enter

异常:
System.ArgumentOutOfRangeException: 该值必须大于或等于零，且必须小于控制台缓冲区在该维度的大小。
参数名: top
实际值是 -15。
   在 System.Console.SetCursorPosition(Int32 left, Int32 top)
   在 Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   在 Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   在 Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   在 Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   在 Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   在 Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   在 Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------

**Comment by hippieZhou** [comment]
Yeah，I alse have this issue. Who can tell me  how to fix it.

```bash
哎呀，出现问题。请在报告此 Bug 时添加以下详细信息。
在 GitHub 上报告: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
上 200 个密钥:
 \ m s - p y t h o n . p y t h o n - 2 0 1 9 . 9 . 3 4 9 1 1 \ p y t h o n F i l e s \ p t v s d _ l a u n c h e r . p y ' Space ' - - d e f a u l t ' Space ' - - c l i e n t ' Space ' - - h o s t ' Space ' l o c a l h o s t ' Space ' - - p o r t ' Space ' 6 1 4 3 7 ' Space ' C : \ U s e r s \ h i p p i e \ D e s k t o p \ d j a n g o g i r l \ s r c \ m a n a g e . p y ' Space ' r u n s e r v e r ' Space ' - - n o r e l o a d ' Space Enter


异常:
System.ArgumentOutOfRangeException: 该值必须大于或等于零，且必须小于控制台缓冲区在该维度的大小。
参数名: top
实际值是 -6。
   在 System.Console.SetCursorPosition(Int32 left, Int32 top)
   在 Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   在 Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   在 Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   在 Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   在 Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   在 Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   在 Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
```

**Comment by daxian-dbw** [comment]
This should have been fixed by #979
The `2.0.0-beta5` version of PSReadLine was released this morning, which contains the fix. Please get it from PowerShellGallary.

**Comment by flashlxy** [comment]
Thank you.




------------------ 原始邮件 ------------------
发件人: "slsters"<notifications@github.com>; 
发送时间: 2019年9月14日(星期六) 凌晨3:50
收件人: "PowerShell/PSReadLine"<PSReadLine@noreply.github.com>; 
抄送: "非对称"<flashlxy@qq.com>; "Author"<author@noreply.github.com>; 
主题: Re: [PowerShell/PSReadLine] System.ArgumentOutOfRangeException: 该值必须大于或等于零，且必须小于控制台缓冲区在该维度的大小。 (#1041)




This should have been fixed by #979
 The 2.0.0-beta5 version of PSReadLine was released this morning, which contains the fix. Please get it from PowerShellGallary.
